readme.md

```diff
@@ -50,14 +50,62 @@ app.on('ready', () => {
 });
 ```
 
-### Options
+## API
 
-#### showDevTools
+### Default exported function
+
+```js
+electronDebug({ showDevTools = false });
+```
+
+Install keyboard shortcuts and optionally
+activate DevTools on each created `BrowserWindow`.
+
+#### Options
+
+* showDevTools
 
 Type: `boolean`  
 Default: `false`
 
-Show DevTools on startup.
+Show DevTools on each created `BrowserWindow`.
+
+
+### devTools
+
+```js
+electronDebug.devTools(win = undefined);
+```
+
+Toggle DevTools for specified `BrowserWindow` instance or for focused one if not specified.
+
+#### Options
+
+* win
+
+Type: `BrowserWindow`  
+Default: `undefined`
```

Comment:
Do you refer to "Show DevTools on each created `BrowserWindow`.". 
I think this is the behaviour actived by the option: each BW created will trigger devtools open...